CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## angular-sandbox Changelog
```

Comment:
@flesch just a force of habit.  But I do agree it is redundant, will be removed when I address a few other issues @sataroz mentioned